rskj-core/src/main/java/co/rsk/peg/abi/ABICallElection.java

```diff
@@ -61,46 +66,54 @@ public void clear() {
      */
     public boolean vote(ABICallSpec callSpec, RskAddress voter) {
```

Comment:
We should probably log the voter also. Maybe at the beginning of the method

Reply:
Done